src/Core/View.js

```diff
@@ -165,12 +151,6 @@ function _preprocessLayer(view, layer, provider, parentLayer) {
                 providerPreprocessing = Promise.resolve();
```

Comment:
could you rename `providerPreprocessing` in `whenReady`.

Reply:
Done